app/services/storage.js

```diff
@@ -94,14 +111,20 @@ export default Ember.Service.extend(Ember.Evented, {
   },
 
   setupRemoteStorage() {
+    const remoteStorage = this.get('remoteStorage');
```

Comment:
Is there a reason to leave the remoteStorage instance creation in `init()`, if it's not being configured entirely there?